src/components/UserProfile/UserSettings/UserNotificationSettings/index.tsx

```diff
@@ -123,6 +128,22 @@ const UserNotificationSettings: React.FC = () => {
                   />
                 </div>
               </div>
+              <div className="form-row">
+                <label htmlFor="pgpKey" className="text-label">
+                  <span>{intl.formatMessage(messages.pgpKey)}</span>
```

Comment:
```suggestion
                  <span>{intl.formatMessage(messages.pgpKey)}</span>
                  <Badge badgeType="danger">
                      {intl.formatMessage(globalMessages.advanced)}
                    </Badge>
```

I think we should mark this field as "Advanced," similar to the "Enable CSRF Protection" setting.